tests/api/test_category.py

```diff
@@ -12,41 +12,197 @@
 from tests.utils import create_image, create_pdf_file_with_image_ext
 
 
-def test_category_query(user_api_client, product):
+def test_category_query_by_id(user_api_client, product):
     category = Category.objects.first()
-    query = """
-    query {
-        category(id: "%(category_pk)s") {
+    category_pk = graphene.Node.to_global_id("Category", category.pk)
```

Comment:
Maby extract this query to const with input as a `query variable`, if we do it we can have one graphql query for all test modified in this PR.

Reply:
Yes as i wrote on Slack, this is done this way for better readability of TC

Reply:
And will be changed when we agree on API

Reply:
Refactored